runtime/doc/news.txt

```diff
@@ -169,6 +169,8 @@ LSP
   https://microsoft.github.io/language-server-protocol/specifications/specification-current/#workspace_dagnostics
 • Incremental selection is now supported via `textDocument/selectionRange`.
   `an` selects outwards and `in` selects inwards.
+• Diagnostic related information is now shown in
+  |vim.diagnostic.open_float()|.
```

Comment:
This confused me at first ("I thought it already did that?"). I think it makes sense to reference the literal technical name here, `DiagnosticRelatedInformation`.

Also, `vim.diagnostic.open_float()` is not specific to LSP so it's not clear how this new feature is surfaced from LSP-enabled buffers. Do I just run `:lua vim.diagnostic.open_float()` to try it out, or is there a default LSP mapping ?